kpatch-build/kpatch-build

```diff
@@ -215,6 +218,12 @@ find_core_symvers() {
 	[[ -e "$SYMVERSFILE" ]]
 }
 
+# CONFIG_DEBUG_INFO_BTF invokes pahole, which needs to have extended ELF
+# header support (version TBD).
+pahole_btf_version_check() {
+	[[ ! "$(pahole --version 2>/dev/null)" < "vTBD" ]]
+}
```

Comment:
I think pahole will fix this in its next v1.20 release, but does it make sense to universally disable this if CONFIG_DEBUG_INFO_BTF is set?  This would save everyone a little bit of kpatch-build time and I could take out this pahole version check (which may or may not reflect reality if a fix is backported to something like dwarves-1.19-2.el8).

Reply:
Yeah, maybe so